client/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Agroeasy",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.1.1",
+    "react": "^16.4.1",
+    "react-bootstrap": "^0.32.1",
+    "react-dom": "^16.4.1",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "1.1.4",
+    "reactstrap": "^6.1.0"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "proxy":"http://localhost:5000"
+}
```

Comment:
combine this `package.json` with the one in the root folder

Reply:
Is it only this part selected or the entire package.json of contents?

Reply:
there should only be one `package.json` defined per project. Merge the contents of this one with the one in the root folder